perun-wui-core/src/main/java/cz/metacentrum/perun/wui/model/beans/RichUserExtSource.java

```diff
@@ -0,0 +1,38 @@
+package cz.metacentrum.perun.wui.model.beans;
+
+import cz.metacentrum.perun.wui.client.utils.JsUtils;
+
+/**
+ * Extended UserExtSource entity containing the original ues and its email Attribute.
+ *
+ * @author Vojtech Sassmann &lt;vojtech.sassmann@gmail.com&gt;
+ */
+public class RichUserExtSource extends UserExtSource {
```

Comment:
I believe we should have objects referencing only real beans from core in perun-wui-core. Right now, there isn't any UES with attributes. It's probably now necessary for GUI to simply work, but please move it to profile project, if it's not used in another app and we should rework it.

Reply:
I have moved it to profile.